src/main/java/com/taboola/backstage/internal/extensions/BackstageExtensionUtilsImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.taboola.backstage.internal.extensions;
+
+/**
+ * Created by vladi
+ * Date: 3/4/2018
+ * Time: 12:37 AM
+ * By Taboola
+ */
+public class BackstageExtensionUtilsImpl implements BackstageExtensionUtils {
```

Comment:
maybe I miss something - why do we need to create this abstraction and not use directly the EndpointFactory ?

Reply:
@barkamir was thinking of backward compatibility in future releases